src/components/EmptyPage.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React, { useContext } from 'react';
+import { StyleSheet } from 'react-native';
+import { Text } from 'react-native-elements';
+
+import { colors, consts, normalize } from '../config';
+import { OrientationContext } from '../OrientationProvider';
+
+export const EmptyPage = ({ title }) => {
+  const { orientation, dimensions } = useContext(OrientationContext);
+  const needLandscapeStyle =
+    orientation === 'landscape' || dimensions.width > consts.DIMENSIONS.FULL_SCREEN_MAX_WIDTH;
+
+  return (
+    <Text
+      h4
+      h4Style={[styles.titleStyle, needLandscapeStyle && styles.titleStyleLandscape]}
+      containerStyle={[styles.containerStyle, needLandscapeStyle && styles.containerStyleLandscape]}
+    >
+      {title}
+    </Text>
```

Comment:
there are some `Text` components which are the base for every rendered text in the app. good point, that you have found `react-native-elements` already. `RegularText` is something we use often with some different props available. can you try to use some existing text component here?